### ISSUE TITLE
Add the possibility to report servers according to their title/description (rules needed)

### ISSUE THREAD

**Issue by Zughy** [luanti-org/serverlist#42]
## Problem
See https://github.com/minetest/minetest/issues/8710#issuecomment-751237200
Anyone at the moment could create a server with a description such as "[name] should die", "let's burn all the [ethnicity]" etc.

## Solutions
(steps)
1. Discuss and write down a set of rules
2. Implement a report system
3. Have someone moderating it
4. Profit

## Alternatives
Don't, but this can become a huge problem. See the aforementioned comment

## Additional context
MT could feature some sort of 18+ tag for porn and general kinks that don't hurt no one/spread hate. By no means servers with a political vision should be penalised as long as, again, that political vision is not based on hate. For instance, a server called "The Almighty NeoNazis" should be banned, whereas Anarchy servers shouldn't. MT should only cover servers titles and descriptions. If servers admins spread hate, it's not MT responsibility and it should be up to people to expose these toxic elements.

**Comment by sfan5** [comment]
By "add the possibility" do you mean inside the Minetest client?
You can and could always talk to the team if there are any concerns. Adding a report system is a waste of time IMO because issues are extremely rare.

> Discuss and write down a set of rules

There is isn't anything official or written down, but the only rule we enforce so far is pretty much "keep it family friendly" for names/descriptions show on the server list.

> MT should only cover servers titles and descriptions.

I agree that MT should not police contents of servers.
However I wouldn't set this rule into stone. If we receive knowledge of questionable or even illegal things happening, it can make sense to delist a server from the list.

**Comment by Zughy** [comment]
> Adding a report system is a waste of time IMO because issues are extremely rare

How about in the future? If MT becomes popular, these things are more likely to happen and I don't think you core devs would like to be bothered every time with "hey look at this, look at that". It's not a priority now for sure, I'm just trying to imagine when the time spent developing a report system becomes lesser than the time spent dealing with separate reports from separate sources

> the only rule we enforce so far is pretty much "keep it family friendly"

Eventually it'd be nice to have some sort of 18+ filter, maybe hidden by default that could be activated via advanced settings. I'm not into these things, but some are and in the end they're not harming anyone

> If we receive knowledge of questionable or even illegal things happening it can make sense to delist a server from the list

This is tricky, there are a lot of gray areas and one mistake can backfire, hard. I genuinely don't know

**Comment by Calinou** [comment]
I think delisting servers from the list has been done in the past already. At the end of the day, the master server is an online service. We can choose to include or exclude anyone at our own discretion :slightly_smiling_face:

**Comment by Zughy** [comment]
Yes, you can, but providing a few general rules would be better to avoid sounding like dictators. Even just the family friendly thing sfan mentioned

**Comment by jgoffredo** [comment]
I need report a server with drugs and pornography and others offensive items in the first image.

It is "Your Island" server.

**Comment by Zughy** [comment]
Closing due to the lack of interest + previous comments

**Comment by rubenwardy** [comment]
I think the server list website should have some rules and an email address to report issues. You don't need a full reporting system, just a contact address
